### DISCUSSION TITLE
EGL for OpenGL Linux support

### DISCUSSION THREAD

**Discussion by elisemorysc** [google/filament#3912] [Ideas]
Hi,

The Linux version of filament for OpenGL supports only GLX for context. NVIDIA drivers now supports using EGL on Linux which is useful especially when rendering in headless servers since it doesn't requires x support. An added bonus is that this allows filament to render into transparent surfaces which GLX doesn't support.

We have a working version for filament that supports this. Is this interesting for the project? I can submit a PR if yes.

**Comment by romainguy** [comment]
@pixelflinger
